configs/components/echam/echam.yaml

```
@@ -352,6 +352,9 @@ choose_scenario:
                         swflux: 1950swflux
                         volcir: 1950volcir
                         volcsw: 1950volcsw
+        # new scenario NUDG for ECHAM6-wiso nudged simulations (M. Werner, 09/2022)
+        NUDG:
```

To my knowledge, there is no requirement for scenario names to be short, so we can be explicit here. Small level peanuts changes:

```suggestion
        NUDGING:
```